### PR TITLE
ZNA-3958,ZNA-1592

### PR DESCRIPTION
**Pull request by Tejas7194**
Analytics Event Added and Download Deeplink

### CODE REVIEW

File: PluginClasses/Zee5PluggablePlayer.swift [hunk 1]
```diff
@@ -42,8 +43,13 @@ public class Zee5PluggablePlayer: APPlugablePlayerBase, ZPAdapterProtocol {
             return nil
         }
         
-        Zee5DownloadManager.shared.initializeDownloadManger()
+
+        print("pluggablePlayerRemoveInline::Videos : \(videos)")
```
Comment (sborkin) [inline PluginClasses/Zee5PluggablePlayer.swift:47]: please remove logs and comment

File: PluginClasses/Zee5PluggablePlayer.swift [hunk 2]
```diff
@@ -76,6 +82,42 @@ public class Zee5PluggablePlayer: APPlugablePlayerBase, ZPAdapterProtocol {
         
         instance.kalturaPlayerController = playerViewController
         
+        let Dict = items!.first!.extensionsDictionary
```
Comment (sborkin) [inline PluginClasses/Zee5PluggablePlayer.swift:85]: I removed it from here in purpose, why it needed here? this whole init was moved into the playContent()

Comment (Tejas7194) [reply PluginClasses/Zee5PluggablePlayer.swift:85]: Yes I checked That.

File: PluginClasses/Zee5PluggablePlayer.swift (hunk 1 above)
Comment (Tejas7194) [reply PluginClasses/Zee5PluggablePlayer.swift:47]: Logs and Comment ? or that download manager initializer?

**Pull request closed without merging**